### PR TITLE
fix IOOS ngdac links

### PR DESCRIPTION
**Pull request by ocefpaf**
@kbailey-noaa this should fix all the references to ioosngdac in this page.

### CODE REVIEW

File: _docs/index.md [hunk 1]
```diff
@@ -24,10 +24,10 @@ The IOOS Metadata Profile contains dataset attribution guidelines and examples t
 * [IOOS Metadata Profile](https://ioos.github.io/ioos-metadata/)
 
 
-### **IOOS National Glider DAC Wiki**
+### **IOOS National Glider DAC**
 The IOOS National Glider DAC is the official Data Assembly Center (DAC) for all IOOS underwater glider activities.  The Wiki provides glider operators and IOOS data providers technical details on the Glider DAC netCDF data format specifications, information on data provider registration and data set submission processes for contributing real-time and delayed-mode glider data sets to the DAC.  The Glider DAC website includes map-based visualizations, data access links, and information about the Underwater Glider User Group (UG2).
```
Comment (kbailey-noaa) [inline _docs/index.md]: Can you also please update this paragraph to:
The IOOS National Glider DAC is the official Data Assembly Center (DAC) for all IOOS underwater glider activities. The Glider DAC documentation provides glider operators and data providers technical details on netCDF data format specifications, information on data provider registration and data set submission processes for contributing real-time and delayed-mode glider data sets to the DAC. The Glider DAC GitHub is the main repository for glider DAC code and tools, and can be used to submit issues or request enhancements to the Glider DAC. The Glider DAC website includes map-based visualizations, data access links, and information about the Underwater Glider User Group (UG2).

**Review by kbailey-noaa — CHANGES_REQUESTED** [review]
Changes requested for row 28

File: _docs/index.md (hunk 1 above)
Comment (ocefpaf) [reply _docs/index.md]: Done!

**Comment by ocefpaf** [comment]
> Changes requested for row 28

Should be OK now.

Sorry I forgot to comment here, the change was implemented a while back.

**Review by kbailey-noaa — APPROVED** [review]

**Pull request merged**